src/GregClient/GregClient.csproj

```diff
@@ -1,18 +1,24 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+	<Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <PackageId>Greg</PackageId>
+    <Authors></Authors>
+    <Company>Autodesk</Company>
     <ProductVersion>8.0.30703</ProductVersion>
     <SchemaVersion>2.0</SchemaVersion>
     <ProjectGuid>{644207B4-7E7F-474A-952E-3453960D8A01}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Greg</RootNamespace>
     <AssemblyName>Greg</AssemblyName>
-    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
+    <TargetFrameworks>net452;net48;net5.0</TargetFrameworks>
```

Comment:
why target net 5 when its EOL is in a month?!

Reply:
Net6 is definitely the final goal. 

My thinking was to target net50 (even if it is EOL) before we transition to net6 (for Dynamo and related projects) because:
 1. Dotnet6 is not supported on vs2019 - this requires more work on some of our dependencies too (ex. CICD).
 2. Less breaking changes compared to dotnet framework 4.8 (ex a lot of the AppDomain related APIs are gone in dotnet6 but are still around in dotnet5)
 3. Some Dynamo dependencies (ex. ADP) have net5.0 for now. Net6 will probably be scheduled later this year.
 
 I am proposing to target net50 for now (internally at least - no public deliverables are needed), just so that we can start the transitioning work right now... Switching from net5 to net6 would be a lot easier to do.